analysis-pipeline.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# BioBombe Analysis Pipeline
+# Gregory Way, 2018
+#
+# This file will perform the entire BioBombe analysis pipeline. Modules 0, 1, 2, and 3
+# are not performed in this script. Module 0 downloads and processes raw gene expression
+# data from GTEX, TCGA, and TARGET. Module 1 performs a parameter sweep over the neural
+# network models (Denoising Autoencoder and Variational Autoencoder). Module 2 actually
+# performs the serial compression with increasing bottleneck layers. Module 3 builds
+# the real and permuted networks used to interpret the compressed features.
+#
+# The first step in this pipeline is to download the precomputed data output from
+# Module 2
+
+conda activate biobombe
+
+##############################
+# Step 1: Get Data (Module 2)
+##############################
+cd 2.ensemble-z-analysis
+
+python download-results-archive.py
+
+##############################
```

Comment:
I see module 3 is skipped...I thought that you used your permuted network for your stability analysis?